### DISCUSSION TITLE
Menu to activate onMouseOver?

### DISCUSSION THREAD

**Discussion by mikeytown19** [chakra-ui/chakra-ui#2614] [Q&A] [answered]
If this can already be done I'd love to know how, but I'm wondering how I could add a Menu to my project and making it open on being hovered. 

https://chakra-ui.com/docs/overlay/menu#accessing-the-internal-state

**Comment by with-heart** [comment, marked as answer]
Hi @mikeytown19, you could do something like this: https://codesandbox.io/s/mystifying-water-60s0f?file=/src/App.tsx

**Reply by mikeytown19** [reply]
@with-heart ahhh I didn't know about `useDisclosure` yes! thank you so much 👍

**Reply by Chinomso1995** [reply]
what if you are using multiple elements and mapping them. This toggles eeverything
